tests/custom-package-edit-visibility-test.js

```diff
@@ -51,24 +51,10 @@ describeApplication('CustomPackageEditVisibility', () => {
     });
 
     describe('toggling the visiblity toggle', () => {
-      beforeEach(function () {
-        /*
-         * The expectations in the convergent `it` blocks
-         * get run once every 10ms.  We were seeing test flakiness
-         * when a toggle action dispatched and resolved before an
-         * expectation had the chance to run.  We sidestep this by
-         * temporarily increasing the mirage server's response time
-         * to 50ms.
-         * TODO: control timing directly with Mirage
-         */
-        this.server.timing = 50;
```

Comment:
Best part of this PR: we don't have to jump through weird test timing hoops for the toggle switch animations anymore 😀